cvat/apps/engine/mixins.py

```diff
@@ -339,6 +340,7 @@ class CreateModelMixin(mixins.CreateModelMixin):
     def perform_create(self, serializer, **kwargs):
         serializer.save(**kwargs)
         signal_create.send(self, instance=serializer.instance)
+        event_signal_create.send(self, instance=serializer.instance)
```

Comment:
We already have multiple problems in WebHooks app due to own signals. Please look at the issue: https://github.com/opencv/cvat/issues/5631

Use Django DB signals only! They should be enough to track how the server state is changed in most cases.